java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/WindowAggPlan.scala

```diff
@@ -149,50 +148,38 @@ object WindowAggPlan {
     }
 
     // Get repartition keys and orderby key
-    // TODO: Now it does not support columns with the same names because it uses column name for repartition
-    val repartitionColNames = PhysicalNodeUtil.getRepartitionColumnNames(windowAggNode, inputDf).asJava
+    // TODO: Support multiple repartition keys and orderby keys
     val repartitionColIndexes = PhysicalNodeUtil.getRepartitionColumnIndexes(windowAggNode, inputDf)
-    val orderbyColName = PhysicalNodeUtil.getOrderbyColumnName(windowAggNode, inputDf)
-    val orderbyColIndex = PhysicalNodeUtil.getOrderbyColumnIndex(windowAggNode, inputDf)
-    val orderbyColType = inputDf.schema(orderbyColIndex).dataType
+    val percentileColIndex = PhysicalNodeUtil.getOrderbyColumnIndex(windowAggNode, inputDf)
```

Comment:
This is the column from user's SQL "order by", `orderbyColIndex` seems better than `percentileColIndex`.